github/ipfs.yml

```diff
@@ -6651,63 +6520,49 @@ teams:
         - jonnycrunch
         - JustMaier
         - kanej
-        - kinu
         - kk3wong
         - koalalorenzo
         - Kubuxu
         - leshokunin
         - lidel
         - listenaddress
-        - ma82
         - mafintosh
         - magnshen
-        - MAkcanca
         - maparent
         - MarneeDear
         - MichaelMure
-        - moeghashim
         - momack2
         - moul
-        - neogeweb3
         - obo20
         - olizilla
         - parkan
         - pepoospina
-        - pooja
         - Prtfw
-        - ratoshniuk
         - raulk
         - requilence
         - rittme
         - rklaehn
         - roignpar
         - romaric-juniet
         - Schwartz10
-        - ShishKabab
-        - smihaylov
         - Stebalien
         - stefanhans
         - steven004
-        - teh-f00l
         - terichadbourne
         - vasa-develop
         - vojtechsimetka
         - waynewyang
-        - whyrusleeping
         - Wondertan
         - yrliou
-        - zeckli
     privacy: closed
   IPFS Camp 2019 Org Team:
```

Comment:
Remove entire team, will save us noise over time.

Reply:
Noted, I'll remove it in a separate PR.